mmedit/datasets/sr_custom_multiple_gt_dataset.py

```diff
@@ -0,0 +1,75 @@
+import glob
+import os.path as osp
+
+from .base_sr_dataset import BaseSRDataset
+from .registry import DATASETS
+
+
+@DATASETS.register_module()
+class SRCustomMultipleGTDataset(BaseSRDataset):
+    """Custom dataset for video super resolution for recurrent networks.
```

Comment:
Docstring failed to explain `Multiple`.
Besides, This sentence is a bit difficult to understand (... for video super resolution for recurrent networks.)

Reply:
It is explained in line 15: `The dataset loads several LQ (Low-Quality) frames and GT (Ground-Truth) frames.`. There are multiple GT frames.